docs/Specifications.md

```diff
@@ -0,0 +1,75 @@
+# Functional Specifications
+
+Why do we write out our functional specifications? So no one says two days before the first match, "No one told me the robot had to be able to do *that*!"
+
+## Will Do / Can Do Table
+
+As we write our specification and code we will fill out this table.
+
+Category | Specification | Will Do | Chassis Can Do | Comp Can Do
+---------|---------------|---------|----------------|------------
+Move | Move responding to remote human manual control | YES | NO | NO
+Move | Translate and rotate at the same time | YES | NO | NO
+Shoot | Shoot cargo into high goal | YES | NO | NO
+Shoot | Shoot and Move | NO | NO | NO
+Shoot | Check color using color sensor | YES | NO | NO 
+Shoot | Eject opponent cargo at low power| YES | NO | NO
+Auto | Complete an autonomous routine, scoring one to four cargo | YES | NO | NO
+Climb | Climb to Top Bar | YES | NO | NO
+V-SLAM | Localize Ourself on the field | YES | NO | NO
```

Comment:
If we can't do the advanced things like moving and shooting, what's the benefit of localization if our vision system works well?